### DISCUSSION TITLE
Team Lead Change

### DISCUSSION THREAD

**Discussion by Jahnavi-Anand** [Jahnavi-Anand/Hospital_Management_System#77] [Announcements]
Hii

**Comment by Aanidhay** [comment]
![WhatsApp Image 2023-11-18 at 23 28 01_c3d0700f](https://github.com/Jahnavi-Anand/Hospital_Management_System/assets/115068672/dca258eb-ac85-41af-9883-ed326a9ddec9)
Discussed for the team lead change with the team members on 18th Nov, 2023

**Comment by Aanidhay** [comment]
Hello Guys,
As I had discussed in the meet yesterday, now Swarnima Raj Mohan Bisht is your new Team Lead as I will not be able to attend the D-day.
Thanks and Regards
Aanidhay Aggarwal

All the best guys!!!!
